examples/graph/ipynb/mpnn-molecular-graphs.ipynb

```diff
@@ -49,98 +46,98 @@
     "graph neural network implemented in this tutorial see\n",
     "[Neural Message Passing for Quantum Chemistry](https://arxiv.org/abs/1704.01212) and\n",
     "[DeepChem's MPNNModel](https://deepchem.readthedocs.io/en/latest/api_reference/models.html#mpnnmodel)."
-   ]
+   ],
+   "metadata": {
+    "colab_type": "text"
+   }
   },
   {
    "cell_type": "markdown",
-   "metadata": {
-    "colab_type": "text"
-   },
    "source": [
-    "## Setup\n",
-    "\n",
-    "### Install RDKit and other dependencies\n",
-    "\n",
-    "(Text below taken from\n",
-    "[this tutorial](https://keras.io/examples/generative/wgan-graphs/)).\n",
-    "\n",
-    "[RDKit](https://www.rdkit.org/) is a collection of cheminformatics and machine-learning\n",
-    "software written in C++ and Python. In this tutorial, RDKit is used to conviently and\n",
-    "efficiently transform\n",
-    "[SMILES](https://en.wikipedia.org/wiki/Simplified_molecular-input_line-entry_system) to\n",
-    "molecule objects, and then from those obtain sets of atoms and bonds.\n",
-    "\n",
-    "SMILES expresses the structure of a given molecule in the form of an ASCII string.\n",
-    "The SMILES string is a compact encoding which, for smaller molecules, is relatively\n",
-    "human-readable. Encoding molecules as a string both alleviates and facilitates database\n",
-    "and/or web searching of a given molecule. RDKit uses algorithms to\n",
-    "accurately transform a given SMILES to a molecule object, which can then\n",
-    "be used to compute a great number of molecular properties/features.\n",
-    "\n",
-    "Notice, RDKit is commonly installed via [Conda](https://www.rdkit.org/docs/Install.html).\n",
-    "However, thanks to\n",
-    "[rdkit_platform_wheels](https://github.com/kuelumbus/rdkit_platform_wheels), rdkit\n",
-    "can now (for the sake of this tutorial) be installed easily via pip, as follows:\n",
-    "\n",
-    "```\n",
-    "pip -q install rdkit-pypi\n",
-    "```\n",
-    "\n",
-    "And for easy and efficient reading of csv files and visualization, the below needs to be\n",
-    "installed:\n",
-    "\n",
-    "```\n",
-    "pip -q install pandas\n",
-    "pip -q install Pillow\n",
-    "pip -q install matplotlib\n",
-    "pip -q install pydot\n",
-    "sudo apt-get -qq install graphviz\n",
+    "## Setup\r\n",
```

Comment:
Please remove the \r character everywhere.

Reply:
Rather than regenerating the ipynb, I suggest editing the previous ipynb file by hand, since it's just a typo.

Reply:
Thanks a lot, @fchollet. I removed \r character everywhere and fixed the typo in md and py files. Let me know if I did something wrong, I will fix it asap.